test/Microsoft.AspNetCore.Blazor.Build.Test/ReferencedAssemblyFileProviderTest.cs

```diff
@@ -1,128 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using Microsoft.AspNetCore.Blazor.Build.Core.FileSystem;
-using Microsoft.AspNetCore.Blazor.Mono;
-using Mono.Cecil;
-using System;
-using System.IO;
-using System.Linq;
-using Xunit;
-
-namespace Microsoft.AspNetCore.Blazor.Server.Test
-{
-    public class ReferencedAssemblyFileProviderTest
-    {
-        [Fact]
-        public void RootDirContainsOnlyDlls()
-        {
-            var provider = new ReferencedAssemblyFileProvider(
-                "mscorlib",
-                new ReferencedAssemblyResolver(MonoStaticFileProvider.BclFiles, string.Empty));
-            foreach (var item in provider.GetDirectoryContents("/"))
-            {
-                Assert.False(item.IsDirectory);
-                Assert.EndsWith(".dll", item.Name);
-            }
-        }
-
-        [Fact]
-        public void FindsReferencedAssemblyGraphSimple()
-        {
-            var provider = new ReferencedAssemblyFileProvider(
-                "System.Linq.Expressions",
-                new ReferencedAssemblyResolver(MonoStaticFileProvider.BclFiles, string.Empty));
-            var contents = provider.GetDirectoryContents("").OrderBy(i => i.Name).ToList();
-            Assert.Collection(contents,
-                item => { Assert.Equal("/Mono.Security.dll", item.PhysicalPath); },
-                item => { Assert.Equal("/mscorlib.dll", item.PhysicalPath); },
-                item => { Assert.Equal("/System.Core.dll", item.PhysicalPath); },
-                item => { Assert.Equal("/System.dll", item.PhysicalPath); },
-                item => { Assert.Equal("/System.Linq.Expressions.dll", item.PhysicalPath); },
-                item => { Assert.Equal("/System.Xml.dll", item.PhysicalPath); });
-        }
-
-        [Fact]
-        public void FindsReferencedAssemblyGraphRealistic()
```

Comment:
What's the equivalent to this test for the new implementation of dependency graph walking?

Reply:
Fair point, I'll bring the test back